src/backend/joanie/urls.py

```diff
@@ -58,3 +61,39 @@
         ]
         + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
     )
+
+    try:
+        # Try to import `drf_yasg` dynamically as this dependency is installed only
+        # in a development context then configure schema views and routes
+        openapi = import_module("drf_yasg.openapi")
+        views = import_module("drf_yasg.views")
```

Comment:
is there a different with this syntax?
```suggestion
        from drf_yasg import openapi, views
```

Reply:
Indeed it works.